server/configuration.go

```diff
@@ -186,6 +186,17 @@ func (p *Plugin) OnConfigurationChange() error {
 		return errors.Wrap(err, "failed to ensure demo user")
 	}
 
+	botId, ensureBotError := p.Helpers.EnsureBot(&model.Bot{
+		Username:    "demoplugin",
+		DisplayName: "Demo Plugin Bot",
+		Description: "A bot account created by the demo plugin.",
+	})
+	if ensureBotError != nil || botId == "" {
```

Comment:
nit: is it necessary to check `botId`? Presumably the helpers is broken if it returns `"", nil`?

Reply:
Yup. Will fix.